homeassistant/components/ld2410_ble/const.py

```diff
@@ -0,0 +1,9 @@
+"""Constants for the LD2410 BLE integration."""
+
+
+DOMAIN = "ld2410_ble"
+
+DEVICE_TIMEOUT = 30
+LOCAL_NAMES = {"HLK-LD2410B"}
+
+UPDATE_SECONDS = 15
```

Comment:
This doesn't look like its used

Reply:
The linter isn't as powerful as I assumed. Removed.